webapp/src/adapters/solid/FriendManager.ts

```diff
@@ -48,12 +48,4 @@ export default class FriendManager {
         let name = getStringNoLocale(<Thing>friends, FOAF.name);
         return new User(name, webID);
     }
-
-    // public addFriend(user: User): boolean {
```

Comment:
Just a small reminder that, if this functionality is not going to be integrated, the UI must be changed 😄